engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -211,8 +152,8 @@ public void saveScreenshot() {
             return;
         }
 
-        int width = buffers.sceneFinal.width();
-        int height = buffers.sceneFinal.height();
+        int width = buffersManager.getFBO("ocUndistorted").width();
+        int height = buffersManager.getFBO("sceneFinal").height();
```

Comment:
I haven't found any other issue with this PR. If we can address this quickly we might be able to get it merged over the weekend!